bin/install.js

```diff
@@ -222,6 +222,24 @@ writeFileSync(
 
 console.log('✅ Done updating .gitignore');
 
+if (
+  '@upleveled/react-scripts' in projectDependencies &&
+  !projectPackageJson.packageManager
+) {
+  try {
+    const response = await fetch('https://registry.npmjs.org/pnpm/latest');
```

Comment:
let's avoid the fetch and just hardcode it

Reply:
hardcoded the version number
[447871e](https://github.com/upleveled/eslint-config-upleveled/pull/228/commits/447871e576b1b6843f587bc657575fb8f7a668c6)